src/modules/Experiences/ExperiencesList/helpers.js

```diff
@@ -0,0 +1,36 @@
+const getMonthAndYear = (month, year) => {
+  month = month.slice(0, 3);
+  return `${month} ${year}`;
+};
```

Comment:
The name `getMonthAndYear` is a bit vague, and doesn't really match what the code actually does. Maybe a name  more like `formatDate` or `formatMonthYearDate` would be better?